ui/app/components/clients/date-range.hbs

```diff
@@ -3,65 +3,68 @@
   SPDX-License-Identifier: BUSL-1.1
 }}
 
-<div ...attributes>
+<div ...attributes class={{if (and this.version.isCommunity this.startDate this.endDate) "is-absolute"}}>
```

Comment:
since attributes are spread here, this can logic can actually be done on the `Clients::DateRange` component (in case that helps consolidating this conditional to a getter as I suggested below`